pg_yregress/schema.json

```diff
@@ -55,6 +55,14 @@
         "hba": {
           "type": "string",
           "description": "pg_hba.conf config"
+        },
+        "encoding": {
+          "type": "string",
+          "description": "default encoding for database"
+        },
+        "locale": {
+          "type": "string",
+          "description": "default locale for database"
```

Comment:
Perhaps it is worth specifying defaults here (`UTF8/C`), what do you think?

Reply:
I thought about it too but didn't mention because if someone cares about encoding and locale they should specify it anyways even if they match the defaults